[2022-04-13 22:34:56,475] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:21+00:00 [queued]>
[2022-04-13 22:34:56,506] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:21+00:00 [queued]>
[2022-04-13 22:34:56,507] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:34:56,508] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:34:56,508] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:34:56,714] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-03-02 06:00:21+00:00
[2022-04-13 22:34:56,728] {standard_task_runner.py:52} INFO - Started process 10716 to run task
[2022-04-13 22:34:56,745] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-03-02T06:00:21+00:00', '--job-id', '1817', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpfjdwskbx', '--error-file', '/tmp/tmpa32soo5s']
[2022-04-13 22:34:56,751] {standard_task_runner.py:77} INFO - Job 1817: Subtask ingest
[2022-04-13 22:34:57,039] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:21+00:00 [running]> on host 13db72185839
[2022-04-13 22:34:57,301] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T06:00:21+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T06:00:21+00:00
[2022-04-13 22:34:57,304] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:34:57,308] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:34:57,325] {subprocess.py:85} INFO - Output:
[2022-04-13 22:34:57,339] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:34:57,340] {subprocess.py:89} INFO - dags
[2022-04-13 22:34:57,340] {subprocess.py:89} INFO - logs
[2022-04-13 22:34:57,340] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:34:57,341] {subprocess.py:89} INFO - plugins
[2022-04-13 22:34:57,341] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:34:57,342] {subprocess.py:89} INFO - scripts
[2022-04-13 22:34:57,342] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:34:57,343] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:34:57,386] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220302T060021, start_date=20220413T223456, end_date=20220413T223457
[2022-04-13 22:34:57,438] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:34:57,688] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
